app/controllers/api/v1/slides_controller.rb

```diff
@@ -3,6 +3,14 @@
 module Api
   module V1
     class SlidesController < ApplicationController
+      before_action :set_slide, only: %i[ show update destroy ]
+
+      #GET /slides
+      def index 
+        @slide = Slide.all
```

Comment:
Check this out because the slides `belong to` an organization.
```suggestion
        @organization = Organization.find(params[:organization_id])
        @slide = @organization.slides.all
```